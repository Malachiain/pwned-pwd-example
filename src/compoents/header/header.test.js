import React, {Component} from 'react';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import Header from './index'
describe('The Header ',()=>{
    it('It renders the header correctly',()=>{
        const wrapper = shallow(<Header/>)
        expect(toJson(wrapper)).toMatchSnapshot();
    })
})