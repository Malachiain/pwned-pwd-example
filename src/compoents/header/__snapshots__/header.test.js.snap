// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Header  It renders the header correctly 1`] = `
<styled.div>
  <styled.h1>
    Databreach Password Checker
  </styled.h1>
</styled.div>
`;
